package.json

```diff
@@ -1,6 +1,11 @@
 {
   "name": "semver",
   "version": "5.3.0",
+  "keywords":{
```

Comment:
Becuase this is a list of keywords, you want to use `[` vs. `{`, so it should look like:

```js
"keywords": [
...
],
```